server/routes.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = new express.Router();
+const data = require('../data/data');
+const auth = require('../utils/auth');
+const path = require('path');
+const fs = require('fs');
+
+router.get('/', function (req, res) {
+    res.renderMin('index.ejs', data.getData());
```

Comment:
Considering that `compression` and `minifyHTML` middlewares are applied to every response, do you even need to use `renderMin` function?

Reply:
no harm :D

Reply:
But what about speeeeeeeed? :P

Reply:
I'll check this for you. I thinks there was a reason.